package.json

```diff
@@ -58,7 +57,7 @@
     "@appium/types": "^0.x",
     "@semantic-release/changelog": "^6.0.3",
     "@semantic-release/git": "^10.0.1",
-    "@types/bluebird": "^3.5.42",
+    "@types/chai-as-promised": "^7.1.8",
```

Comment:
these types must not imported

Reply:
sorry I don't really understand. I uninstalled the types package for bluebird

Reply:
I remove the package from package.json now